kof-operator/internal/controller/remote-secret/fake_remote_secret_manager.go

```diff
@@ -0,0 +1,33 @@
+package remotesecret
```

Comment:
Usually the package name is the same as folder
```suggestion
package istio
```